### PR TITLE
Fix clusterSelector default value

### PR DESCRIPTION
**Pull request by serngawy**

### CODE REVIEW

File: docs/policygenerator-reference.yaml [hunk 1]
```diff
@@ -118,9 +118,27 @@ policyDefaults:
...
+    # To specify placement with advance expression specify as example below
+    #   labelSelector:
+    #     matchExpressions:
+    #       - key: provider
+    #         opertaot: NotIn
+    #         values:
+    #          - "cloud"
+    #       - key: done
+    #         opertaot: NotExist
     labelSelector: {}
```
Comment (dhaiducek) [inline docs/policygenerator-reference.yaml:142]:
```suggestion
    # To specify a placement, specify key:value pair cluster label selectors or the full LabelSelector for the desired cluster
    # label selectors. (See placementPath to specify an existing file instead.)
    # For example, to specify a placement using matchExpressions:
    #   labelSelector:
    #     matchExpressions:
    #       - key: provider
    #         operator: NotIn
    #         values:
    #          - "cloud"
    labelSelector: {}
```

File: docs/policygenerator-reference.yaml [hunk 2]
```diff
@@ -118,9 +118,27 @@ policyDefaults:
...
+    # To specify placement rule with advance expression specify as example below
+    #   clusterSelector:
+    #     matchExpressions:
+    #       - key: provider
+    #         opertaot: NotIn
+    #         values:
+    #          - "cloud"
+    #       - key: done
+    #         opertaot: NotExist
     clusterSelector: {}
```
Comment (dhaiducek) [inline docs/policygenerator-reference.yaml:130]:
```suggestion
    # To specify a placement rule, specify key:value pair cluster selectors or the full LabelSelector for the desired cluster
    # selectors. (See placementRulePath to specify an existing file instead.)
    # For example, to specify a placement rule using matchExpressions:
    #   clusterSelector:
    #     matchExpressions:
    #       - key: provider
    #         operator: NotIn
    #         values:
    #          - "cloud"
```

**Review by dhaiducek — CHANGES_REQUESTED** [review]
Thanks for the bug fix and doc update! I have some minor wording suggestions.

File: docs/policygenerator-reference.yaml (hunk 1 above)
Comment (serngawy) [reply docs/policygenerator-reference.yaml]: done

File: docs/policygenerator-reference.yaml (hunk 2 above)
Comment (serngawy) [reply docs/policygenerator-reference.yaml]: okay, done

**Review by mprahl — APPROVED** [review]
Thanks!

**Pull request merged**